test/test_duplicate_consistency.py

```diff
@@ -9,30 +9,31 @@
 import yaml
 
 def test_duplicate_consistency():
+    helpers.skip_windows()
```

Comment:
This function is a decorator.

Reply:
I don't understand what that means.

Reply:
Ah, it means you can use it as the following:

```python
@helpers.skip_windows()
def test_common():
```

Reply:
I don't think it will skip windows if it's not used as a decorator.